frontend/src/components/common/SortMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import {
+  Flex,
+  Button,
+  Radio,
+  VStack,
+  Text,
+  RadioGroup,
+  Popover,
+  PopoverTrigger,
+  PopoverBody,
+  PopoverContent,
+  PopoverFooter,
+  Divider,
+  HStack,
+} from "@chakra-ui/react";
+import { FilterOptionsIcon } from "../../assets/icons";
+
+interface SortMenuProps {
+  tableProperties: string[];
+  onSortProperty: React.Dispatch<React.SetStateAction<string>>;
+  onSortOrder: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const SortMenu = ({
+  tableProperties,
+  onSortProperty,
+  onSortOrder,
+}: SortMenuProps): React.ReactElement => {
+  const [sortProperty, setSortProperty] = React.useState("firstName");
+  const [sortOrder, setSortOrder] = React.useState("ascending");
+
+  const propertyList = (
+    <RadioGroup
+      color="blue.300"
+      onChange={(e) => setSortProperty(e)}
+      value={sortProperty}
+    >
+      <VStack alignItems="left" gap={1}>
+        <Text pb="2" textStyle="link">
+          Property
+        </Text>
+        {tableProperties.map((property, index) => (
+          <Radio defaultChecked={index === 1} key={index} value={property}>
```

Comment:
could also do the following to adjust the UI
```suggestion
          <Radio defaultChecked={index === 1} key={index} value={property} style={{ textTransform: "capitalize" }} >
```

Reply:
OH beautiful! Will add, thank you!

Reply:
Just kidding. I think `Radio` is not technically a text component, so doesn't seem like applying the `textTransform` attribute works here. Options here: (1) leave code as is (2) wrap the children text in a `Text` object and apply the `textTransform`. Which do you think is cleaner here?

Reply:
ohh hmmm we can just leave as is then

though i noticed that "firstName" is now displayed as "FirstName" instead of "First Name" but ig it's fine lol cause splitting that might complicate the code

Reply:
yeah - i think when we change it to `Name` we won't have to worry about that anyway!